src/lambdas/initiateAthenaQuery/handler.ts

```diff
@@ -1,46 +1,114 @@
 import { SQSEvent } from 'aws-lambda'
-import { confirmAthenaTable } from '../../sharedServices/athena/confirmAthenaTable'
-import { createQuerySql } from '../../sharedServices/athena/createQuerySql'
+import { confirmAthenaTable } from './confirmAthenaTable'
+import { createQuerySql } from './createQuerySql'
+import { startQueryExecution } from './startQueryExecution'
 import { getQueryByZendeskId } from '../../sharedServices/dynamoDB/dynamoDBGet'
+import { updateQueryByZendeskId } from '../../sharedServices/dynamoDB/dynamoDBUpdate'
 import { updateZendeskTicketById } from '../../sharedServices/zendesk/updateZendeskTicket'
+import { CreateQuerySqlResult } from '../../types/athena/createQuerySqlResult'
+import { StartQueryExecutionResult } from '../../types/athena/startQueryExecutionResult'
+import { ConfirmAthenaTableResult } from '../../types/athena/confirmAthenaTableResult'
 
 export const handler = async (event: SQSEvent): Promise<void> => {
   console.log('Handling Athena Query event', JSON.stringify(event, null, 2))
 
+  const zendeskId = confirmEvent(event)
+
+  const athenaTable = await confirmAthenaTable()
+
+  await checkAthenaTableExists(athenaTable, zendeskId)
+
+  const requestData = await getQueryByZendeskId(zendeskId)
+
+  const querySql = createQuerySql(requestData)
+
+  await confirmQuerySqlGeneration(querySql, zendeskId)
+
+  const queryExecutionDetails = await startQueryExecution(querySql)
+
+  await confirmQueryExecution(queryExecutionDetails, zendeskId)
+
+  return
+}
+
+const confirmEvent = (event: SQSEvent): string => {
```

Comment:
Minor thing, but you could just call this `retrieveZendeskIdFromEvent` for clarity

Reply:
Done